docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,78 @@
 
 ## Feature Gate
 
-Expand CSI Volumes feature is promoted to beta in 1.16 so it is enabled by default. For Kubernetes release before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
+Expand CSI Volumes feature was promoted to beta in kubernetes 1.16, therefore it is enabled by default. For Kubernetes releases before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
 
 ## Sidecar Container
 
-An external-resizer sidecar container implements the logic for watching the Kubernetes API for Persistent Volume claim edits and issuing ControllerExpandVolume RPC call against a CSI endpoint and updating PersistentVolume object to reflect new size.
+An external-resizer sidecar container implements the logic of watching the Kubernetes API for Persistent Volume claim edits, issuing the ControllerExpandVolume RPC call against a CSI endpoint and updating the PersistentVolume object to reflect the new size. This container has already been deployed for you as a part of the vsphere-csi-controller pod.
 
-The resize sidecar container will be deployed in the vsphere-csi-controller pod.
+## Requirements
 
-If EXPAND_VOLUME node capability is implemented, NodeExpandVolume will be called by Kubelet on the node if NodeExpansionRequired is set to true in ControllerExpandVolumeResponse.
+If you are using the TKG cluster service and your environment adheres to the required kubernetes and vSphere CSI driver versions mentioned above skip this section and directly proceed to the `Expand PVC Example` section below to use this feature.
 
-## RBAC Rules
+However, in order to try out this feature using the vanilla kubernetes driver, modify the RBAC rules and the StorageClass definition as mentioned below in your environment.
 
-The following  RBAC rules need to be added to support volume expansion:
+### RBAC Rules
 
-```bash
+Add the `patch` privilege to `persistentvolumes` resource and `update`, `patch` privileges to `persistentvolumeclaims/status` resource.
+
+```yaml
 - apiGroups: [""]
     resources: ["persistentvolumes"]
-    # NOTE: added "patch"
     verbs: ["get", "list", "watch", "update", "create", "delete", "patch"]
 - apiGroups: [""]
-    # NOTE: added "persistentvolumeclaims/status"
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
 ```
 
-## StorageClass
+### StorageClass
 
-In the StorageClass, set allowVolumeExpansion to true.
+Create a new StorageClass or edit the existing StorageClass to set `allowVolumeExpansion` to true.
```

Comment:
add a note that this will be set automatically in tkg based on the vsphere version and not recommended to change to true

Reply:
This is a subsection to the Requirements section above. I have already mentioned that this is needed only for Vanilla cluster and not for TKGS.